client/components/domains/registrant-extra-info/fr-form.jsx

```diff
@@ -277,12 +276,12 @@ class RegistrantExtraInfoFrForm extends React.PureComponent {
 
 	renderOrganizationFields() {
 		const translate = this.props.translate;
-		const extra = this.props.contactDetails.extra;
+		const { extra } = this.props.contactDetails;
 		const {
 			registrantVatId,
 			sirenSiret,
-			trademarkNumber,
-		} = { ...emptyValues, ...extra };
+			trademarkNumber
+		} = defaults( extra, emptyValues );
```

Comment:
> Destructuring should deep clone always.

I disagree 😁 but `defaults` _is_ a little weird.

Note that you can always fall back to the low-tech solution:

```js
const {
  countryOfBirth = emptyValues.countryOfBirth,
  placeOfBirth = emptyValues.placeOfBirth,
} = this.props.contactDetails.extra;
```

though that is quite tedious. Also, I noticed that all values in `emptyValues` are the empty string (`''`); do we expect different values to be added to that object? If not, we could just work with the fact that all these fields are strings that default to empty string:

`const { foo = '', bar = '' } = this.props.contactDetails.extra;`

That said, I can't help but think that we have mechanisms in place for this — notably, React's `defaultProps`. If these values weren't folded under `contactDetails.extra` (maybe a new component, under the current one, could deal with the props?) this would work:

```js
class Hello extends Component {
  static defaultProps = {
    countryOfBirth: '',
    placeOfBirth: '',
  }